app/shared/resource-type-filter/ResourceTypeFilterContainer.js

```diff
@@ -32,7 +32,8 @@ export class UnconnectedResourceTypeFilterContainer extends Component {
 
   render() {
     return (
-      <div>
+      <div className="resource-type-filter-container">
+        <h6>Näytä</h6>
```

Comment:
I guess this could be translated, but I'm not actually sure if this title should be part of this component or actually part of its parent. What would be the english string?